README.md

```diff
@@ -56,11 +56,11 @@ To accomodate different timezones, we schedule community calls at different time
 
 Use the upcoming calendar events below to add the community call events to your calendar.
 
-- Wednesday Jan 11<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221221T09&p1=234&msg=Dapr+Community+Call)
-- Wednesday Jan 25<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221221T09&p1=234&msg=Dapr+Community+Call)
-- Wednesday Feb 8<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221221T09&p1=234&msg=Dapr+Community+Call)
-- Wednesday Feb 22<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221221T09&p1=234&msg=Dapr+Community+Call)
-- Wednesday March 8<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20221221T09&p1=234&msg=Dapr+Community+Call)
+- Wednesday Jan 25<sup>th</sup> 9:00am Pacific Time (PST) - [See it in your local time and add to your calendar](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230125T09&p1=234&msg=Dapr+Community+Call)
```

Comment:
We are forgetting to update the URL. See how I had to fix the previously scheduled calls too.

Reply:
Yes, this was cut and pasted. I will reminder @nyemade-uversky on this.